src/config.json

```diff
@@ -13834,6 +13834,8 @@
     "ff9.io",
     "safemoon-dividend.com",
     "sfar.io",
+    "optimism.trade",
+    "cowswap.sale"
     "metamask-wallet-security.web.app",
     "optimism.trade"
```

Comment:
@davidwgrossman FYI, `optimism.trade` is now twice in the list

Reply:
Thanks for the heads up. I removed it in a follow-up PR